src/main/java/Listing.java

```diff
@@ -0,0 +1,37 @@
+public class Listing {
+
+  String title;
+  Boolean isDone = false;
+
+  public Listing(String s) {
+    this.title = s;
+  }
+
+  public void complete() {
+    isDone = true;
+  }
+
+  public String doneness() {
+    if (isDone) {
+      return "[✓]";
+    } else {
+      return "[✗]";
+    }
+  }
+
+  public void checkDoneness(String s) {
+    if (s.equals("1")) {
+      this.isDone = true;
+    } else {
+      this.isDone = false;
+    }
+  }
```

Comment:
It seems like doneness should be some variation of getDoneness and checkDoneness should be some variation of setDoneness? Please correct me if I am wrong

Reply:
Yes you're right. I have changed it accordingly.